src/Features/CSharp/Portable/UseLocalFunction/CSharpUseLocalFunctionCodeFixProvider.cs

```diff
@@ -104,6 +116,16 @@ public override Task RegisterCodeFixesAsync(CodeFixContext context)
                 editor.RemoveNode(lambdaStatement);
             }
 
+            if (!notDirectInvocationUsages.IsEmpty)
+            {
+                foreach (var usage in notDirectInvocationUsages)
+                {
+                    editor.ReplaceNode(
+                        usage.Parent, 
+                        SyntaxFactory.InvocationExpression(usage.Expression, (usage.Parent as InvocationExpressionSyntax).ArgumentList));
```

Comment:
cleaner to do: (usage.Parent as InvocationExpression).WithExpression(usage.Expression).WithTriviaFrom(usage.Parent)